### ISSUE TITLE
bring it baaaaaaaack

### ISSUE THREAD

**Issue by yaikss** [phlntn/emojibuilder#38]

**Comment by beniter** [comment]
@awifb

Hi there! Spent the last month-ish trying to figure out the issue and got some help to figure it out! After creating the assets as outlined in Issue *8, you need to edit the path created in componentDidMount() in /src/App.js. You need to add "emojibuilder/" without the quotation marks at the front of the definition of the url contained within the promise (given that that's what your master directory is called). That looks like this for me: 

    return Promise.all(assets.map(asset => {
      return new Promise((resolve, reject) => {
        const url = `emojibuilder/${config.assetBasePath}/${asset.name}.png?v4`;

**Comment by alexvoz** [comment]
> @beniter 

Do you have assets?

**Comment by beniter** [comment]
> > @beniter
> 
> Do you have assets?

Yeah, I just used the ones from OpenMoji and Twemoji (Xmoji now I guess) and edited each piece of them into vectors as recommended in Issue #8
